### DISCUSSION TITLE
Windows download flagged as having "Trojan-PSW.Agent" by VirusTotal

### DISCUSSION THREAD

**Discussion by cbourner** [spacedriveapp/spacedrive#2767] [General]
I have hesitated to install on Windows after performing this check.  Is this a real issue?

**Comment by cbourner** [comment]
Evidence of check attached here.

![Screenshot 2024-10-13 230329](https://github.com/user-attachments/assets/67696abc-843e-425d-a483-8bbdaf58b731)

**Reply by bluszcz** [reply]
Do you have URL to the check?

**Reply by HeavenVolkoff** [reply]
Hey, developer here. I did a manual retest with VirusTotal, and it seems we do actually have 2/62 reports on VirusTotal:
https://www.virustotal.com/gui/file/f5ff01d19db21489dbdfe79fa96225b4d07f62b5cb5fb89e97eae2c8631e2148

I believe these are false positives. From a quick internet search, I found other OSS software being incorrectly reported as a `Trojan-PSW.Agent` from Ikarus. I will try to email them notifying of the false positive. About the Google `detected` status, I have no idea, but will try looking into it too.

https://github.com/instaloader/instaloader/issues/823
https://github.com/pyrevitlabs/pyRevit/issues/1638

If you are still uncertain about our software, I just want to clarify that our official releases are built on isolated GitHub task runners and the whole process can be publicly analyzed here:
https://github.com/spacedriveapp/spacedrive/actions/workflows/release.yml

Just click on the action specific to the release you have downloaded and choose your OS/Arch combination, and you will be presented with the full logs of that build. Also, the installer is uploaded directly from the action to GitHub's release:
https://github.com/spacedriveapp/spacedrive/blob/main/.github/workflows/release.yml#L132-L148

Last case scenario, the app is fully open source, so you can also build it yourself, it should be reasonably easy if you have basic CLI/terminal experience, just follow the instructions here: https://github.com/spacedriveapp/spacedrive/blob/main/CONTRIBUTING.md#making-changes-locally
